### DISCUSSION TITLE
Shortcut Key on computer keyboard to STOP the loop playing?

### DISCUSSION THREAD

**Discussion by VTM09** [hydrogen-music/hydrogen#1959] [Hydrogen on Windows]
Hi,
I'm using the SPACEBAR on the computer keyboard to Play/Pause the loop playing in Hydrogen. I wonder what key to PLAY/STOP instead of pause? I couldn't find. Or is there anyway to just disable the PAUSE function but just PLAY/STOP. The reason I'm looking for this option, because I use the foot pedal which I can assign the Spacebar key in. While I playing drum live along with the loop from Hydrogen, I can just tap on the foot pedal to PLAY/STOP; meaning when I STOP, it will go back to the first beat. the PAUSE isn't work because it stops at whatever beat playing. If anyone got the tricks or idea, please share. Thanks so much!

**Comment by theGreatWhiteShark** [comment]
Hey @VTM09 ,

> I wonder what key to PLAY/STOP instead of pause?

There is no shortcut for this particular action yet. You could combine <kbd>Space</kbd> with <kbd>Backspace</kbd>, which is locating the current playback position to the start of the song or pattern (depending whether you are in pattern or song mode). (In future version 1.3.0 of Hydrogen all keyboard shortcuts can be altered and different as well as multiple actions can be assigned to them. But there is still a long way to go and I can't tell when this one will be releases.)

But especially for live I would highly recommend controlling Hydrogen using a MIDI device. Maybe you foot pedal is supporting this already? You could map its MIDI signal sent to the PLAY/STOP action in Preferences > Midi.

The major downside of using keyboard instead of MIDI is that Hydrogen needs focus all the time (it has to be the selected window) and some of its UI elements could eat up the event. E.g. when accidentally clicking spin box or another widget you can write text in <kbd>Space</kbd> will not pause Hydrogen anymore but just insert space in text.
